### PR TITLE
Section about encrypted data in DID docs

### PR DESCRIPTION
**Pull request by rhiaro**
Security Considerations section with the caveats about using encrypted data in DID Documents. Addresses concerns in #25

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/228.html" title="Last updated on Mar 23, 2020, 3:41 PM UTC (e9f4825)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/228/93bc34c...e9f4825.html" title="Last updated on Mar 23, 2020, 3:41 PM UTC (e9f4825)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2847,6 +2847,28 @@ <h2>
...
     </section>
 
+    <section>
+      <h2>
+Encrypted Data in DID Documents
+      </h2>
+      <p>
+DID documents are typically publicly available. Implementers can assume that any
+encrypted data placed in a DID document will eventually be made available in clear
+text to the same audience to which the encrypted data is available.
```
Comment (davidlehn) [inline index.html]: Perhaps change "can assume" and "will eventually" to "should assume" and "may eventually".  Does the text need to briefly explain why?

File: index.html [hunk 2]
```diff
@@ -2847,6 +2847,28 @@ <h2>
         </p>
     </section>
 
+    <section>
+      <h2>
+Encrypted Data in DID Documents
+      </h2>
+      <p>
+DID documents are typically publicly available. Implementers can assume that any
```
Comment (msporny) [inline index.html]:
```suggestion
DID documents are typically publicly available. Encryption algorithms have been 
known to fail due to advances in cryptography and computing power. 
Implementers are advised to assume that any
```

File: index.html [hunk 3]
```diff
@@ -2847,6 +2847,28 @@ <h2>
...
+encrypted data placed in a DID document will eventually be made available in clear
```
Comment (msporny) [inline index.html]:
```suggestion
encrypted data placed in a <a>DID document</a> might eventually be made available in clear
```

File: index.html [hunk 4]
```diff
@@ -2847,6 +2847,28 @@ <h2>
...
+text to the same audience to which the encrypted data is available.
+      </p>
+      <p>
+Encrypting all or parts of DID documents is <em>not</em> an appropriate means to protect data
+in the long term. Similarly, placing encrypted data in DID documents is not an appropriate
+means to include personally identifiable information in DID documents.
+      </p>
+      <p>
+Given the caveats above, if encrypted data is included in a DID document, it should not
```
Comment (msporny) [inline index.html]:
```suggestion
Given the caveats above, if encrypted data is included in a DID document, implementers are advised to not
```

File: index.html [hunk 5]
```diff
@@ -2847,6 +2847,28 @@ <h2>
...
+be encrypted with the public keys of entities that do not wish to be correlated with the
```
Comment (msporny) [inline index.html]:
```suggestion
encrypt with the public keys of entities that do not wish to be correlated with the
```

File: index.html [hunk 6]
```diff
@@ -2847,6 +2847,28 @@ <h2>
...
+DID.
```
Comment (msporny) [inline index.html]:
```suggestion
<a>DID</a>.
```

File: index.html [hunk 7]
```diff
@@ -2847,6 +2847,30 @@ <h2>
...
+DID documents are typically publicly available. Encryption algorithms have been 
+known to fail due to advances in cryptography and computing power. 
+Implementers are advised to assume that any
+encrypted data placed in a <a>DID document</a> might eventually be made available in clear
+text to the same audience to which the encrypted data is available.
+      </p>
+      <p>
+Encrypting all or parts of DID documents is <em>not</em> an appropriate means to protect data
+in the long term. Similarly, placing encrypted data in DID documents is not an appropriate
+means to include personally identifiable information in DID documents.
```
Comment (davidlehn) [inline index.html]: A bit redundant.
```suggestion
means to include personally identifiable information.
```

File: index.html (hunk 1 above)
Comment (davidlehn) [reply index.html]: This looks resolved.

**Pull request merged**